ims/einstein/operations.py

```diff
@@ -212,11 +212,10 @@ def shutdown(self):
 
     # Provisions from HIL and Boots the given node with given image
     @log
-    def provision(self, node_name, img_name, network, nic):
+    def provision(self, node_name, img_name, nic):
```

Comment:
why nic?

Reply:
in bmi-config.cfg we can pass it - anyways the nic used for provisioning is different from nic name in HIL